services/apps/alcs/src/alcs/notification/notification-submission/notification-submission.controller.ts

```diff
@@ -64,4 +64,29 @@ export class NotificationSubmissionController {
     );
     return this.get(fileNumber, req);
   }
+
+  @UserRoles(...ANY_AUTH_ROLE)
+  @Patch('/:fileNumber')
+  async update(
+    @Param('fileNumber') fileNumber: string,
+    @Body('contactEmail') contactEmail: string,
+    @Req() req,
+  ) {
+    if (!contactEmail) {
```

Comment:
Do you need to make sure that the contactEmail is not a white space? Something like contactEmail.trim() ?

Reply:
Its easy to add, might as well :)

Reply:
Done!